Lib/asyncio/selector_events.py

```diff
@@ -341,13 +341,15 @@ def remove_writer(self, fd):
         self._ensure_fd_no_transport(fd)
         return self._remove_writer(fd)
 
-    async def sock_recv(self, sock, n):
+    async def sock_recv(self, sock, n, _warn=warnings.warn):
```

Comment:
Why are you adding new `_warn` parameters to methods in this file? That's not necessary.

Reply:
Fixed